doc/build-unix.md

```diff
@@ -134,7 +133,7 @@ GUI dependencies:
 
 If you want to build bitcoin-qt, make sure that the required packages for Qt development
 are installed. Qt 5 is necessary to build the GUI.
-To build without GUI pass `--without-gui`.
+To build with GUI pass `-DBUILD_GUI=ON`.
 
 To build with Qt 5 you need the following:
 
```

Comment:
"Once these are installed, they will be found by configure and a bitcoin-qt executable will be
built by default."

This is no longer true?

Reply:
So this being the case, shouldn't we delete the line: "Once these are installed, they will be found by configure and a bitcoin-qt executable will be built by default."

Reply:
Thanks! Deleted.